### PR TITLE
Added Santander Cycles to amenity=bicycle_rental

### PR DESCRIPTION
**Pull request by peternewman**
A rehash of #3527 now no names are allowed.

### CODE REVIEW

**Comment by peternewman** [comment]
@rskedgell @bhousel do either/both of you want to review this?

File: brands/amenity/bicycle_rental.json [hunk 1]
```diff
@@ -159,6 +159,21 @@
         "name": "OV-fiets"
       }
     },
+    {
+      "displayName": "Santander Cycles",
+      "id": "santandercycles-fa5175b",
+      "locationSet": {"include": ["gb"]},
+      "matchNames": ["barclays bikes", "barclays cycle hire", "boris bike", "boris bikes"],
```
Comment (peternewman) [inline brands/amenity/bicycle_rental.json:166]: Do these work on brand? Hopefully they can be made to do so if they don't currently?

Comment (bhousel) [reply brands/amenity/bicycle_rental.json:166]: I'm not sure what you mean?

**Comment by bhousel** [comment]
Thanks @peternewman 👍 
I think this looks ok, but I'll squash away a lot of the work-in-progress commits, and remove that stuff in the travis file.

File: brands/amenity/bicycle_rental.json (hunk 1 above)
Comment (peternewman) [reply brands/amenity/bicycle_rental.json:166]: Sorry, I mean do matchNames just work on a name field or will they also cover brands, or is that a question for iD and other consumers?

i.e. if I type in Boris Bike will if offer Santander Cycles?

**Comment by peternewman** [comment]
> I think this looks ok, but I'll squash away a lot of the work-in-progress commits,

Fair enough

> and remove that stuff in the travis file.

It will only do anything if the build fails, but up to you obviously.

Unfortunately from my brief experience of JSON validation they do seem to be rather good at giving you a rather vague and incomprehensible error message when they fail.

File: brands/amenity/bicycle_rental.json (hunk 1 above)
Comment (bhousel) [reply brands/amenity/bicycle_rental.json:166]: > i.e. if I type in Boris Bike will if offer Santander Cycles?

Yes, the brand tag is considered an alternate name also.

**Pull request merged**